pyuvdata/utils.py

```diff
@@ -34,6 +34,27 @@ def _str_to_bytes(s):
     def _bytes_to_str(b):
         return b.decode('utf8')
 
+# polarization constants
+# XXX is the 'p' capitalized or not?
```

Comment:
I can be persuaded on the capitalization issue, but I lean slightly toward a lower case p. I think it is capitalized in the dict for the purposes of the `polstr2num` function that compares the keys to an input string that is called with `.upper()`, although interestingly `.lower()` is used in the `jstr2num` function. Let's pick a capitalization scheme and then make the functions do the comparisons properly.